compiler/rustc_target/src/spec/mod.rs

```diff
@@ -3412,7 +3409,6 @@ impl Target {
         key!(main_needs_argc_argv, bool);
         key!(has_thread_local, bool);
         key!(obj_is_bitcode, bool);
-        key!(forces_embed_bitcode, bool);
         key!(bitcode_llvm_cmdline);
```

Comment:
LLVM still leave some code related to `llvmcmd`, but I believe this will also be removed in the future.